jsonpb/jsonpb_test.go

```diff
@@ -684,7 +731,35 @@ func (m *dynamicMessage) MarshalJSONPB(jm *Marshaler) ([]byte, error) {
 	return []byte(m.rawJson), nil
 }
 
-func (m *dynamicMessage) UnmarshalJSONPB(jum *Unmarshaler, json []byte) error {
-	m.rawJson = string(json)
+func (m *dynamicMessage) UnmarshalJSONPB(jum *Unmarshaler, js []byte) error {
+	m.rawJson = string(js)
 	return nil
 }
+
+func (m *dynamicMessage) Marshal() ([]byte, error) {
```

Comment:
Is this necessary?  If field is exported, I think it'll marshal and unmarshal correctly.  I'm ok with this, just curious though.